near/nep141-locker/src/lib.rs

```diff
@@ -963,6 +960,27 @@ impl Contract {
             required_balance = required_balance.saturating_add(ONE_YOCTO);
         }
 
+        if transfer_message.fee.native_fee.0 > 0 {
+            let native_token_id = self.get_native_token_id(transfer_message.get_origin_chain());
```

Comment:
Theoretically, I can perform a transfer from NEAR to NEAR via Omni Transfer, and in that case, the native_token would be NEAR, which cannot be minted. We don’t have any checks in place to prevent this.

Reply:
Actually in this case you won't be able to provide a proof, because we don't have a Near prover.
anyway, has been added a check here 18ab2d0c86eefc6c3a4a75632f032b215ec23322